@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgVehicles.hpp

```diff
@@ -491,11 +491,12 @@ class CfgVehicles
 			class HitHull
 			{
 				armor = 999;
+				minimalHit=0.050000001;
 				material = 51;
 				name = "fuselage";
 				passthrough = 1;
 				visual = "skin_fuse";
-				explosionShielding = 3;
+				explosionShielding = 999;
```

Comment:
Where was this value pulled from?

Reply:
values are derived from vanilla blackfoot